benchmarks/apache-spark/src/main/scala/org/renaissance/apache/spark/SparkUtil.scala

```diff
@@ -0,0 +1,53 @@
+package org.renaissance.apache.spark
+
+import java.io.FileOutputStream
+import java.nio.file.{Files, Path, Paths}
+
+import org.apache.commons.io.IOUtils
+import org.apache.spark.{SparkConf, SparkContext}
+import org.renaissance.RenaissanceBenchmark
+
+trait SparkUtil {
+
+  val portAllocationMaxRetries: Int = 64
+
+  val winUtils = "/winutils.exe"
+
+  def setUpSparkContext(dirPath: Path, thread_count: Int): SparkContext = {
+    setUpHadoop(dirPath)
+    val conf = new SparkConf()
+      .setAppName(RenaissanceBenchmark.kebabCase(this.getClass.getSimpleName))
+      .setMaster(s"local[$thread_count]")
+      .set("spark.local.dir", dirPath.toString)
+      .set("spark.port.maxRetries", portAllocationMaxRetries.toString)
+      .set("spark.sql.warehouse.dir", dirPath.resolve("warehouse").toString)
+    val sc = new SparkContext(conf)
+    sc.setLogLevel("ERROR")
+    sc
+  }
+
+  def tearDownSparkContext(sc: SparkContext): Unit = {
+    if (sc != null) {
+      sc.stop()
+    }
+  }
+
+  /* For Spark version on renaissance (2.0.0) the Hadoop version is 2.2.0
+   For Windows, the binary zip is not included in the dependencies
```

Comment:
Minor: please use the

```
/**
 * Asterisk at every line.
 */
```

style, to be consistent.

Reply:
Fixed